keel-slack/src/main/kotlin/com/netflix/spinnaker/keel/slack/handlers/GitDataGenerator.kt

```diff
@@ -160,6 +156,33 @@ class GitDataGenerator(
     }
   }
 
+  /**
+   * This is for generating delivery config commit updates.
+   *
+   * generateCommitInfo will create a slack section blocks, which looks like:
+   * "App: keel
+   *  Commit by @emburns
+   *   Update README.md"
+   *
+   */
+  fun generateCommitInfoNoArtifact(sectionBlockBuilder: SectionBlockBuilder,
+                                   application: String,
+                                   imageUrl: String,
+                                   altText: String,
+                                   gitMetadata: GitMetadata): SectionBlockBuilder {
+
+    with(sectionBlockBuilder) {
+      markdownText("*App:* $application\n" +
+        "Commit by @${gitMetadata.author}\n " +
```

Comment:
Just a heads-up that, if you want tagging to work in Slack, you can't just use the commit author here, you need to call the Slack API to translate the user email to a Slack username, like here: https://github.com/spinnaker/keel/pull/1864/files#diff-a3f4013f961d345a64b48d91be4d7f2f7c3952522e05f850d5b0ca8c30fb6d12R31

Reply:
oh right, good catch. I got thrown off because in my local tests this was working, but of course I was passing my slack handle and not the email.